docs/EmbeddedAppIOS.md

```diff
@@ -161,6 +161,21 @@ In root directory, we need to start React Native development server.
 
 This command will start up a React Native development server within our CocoaPods dependency to build our bundled script. The `--root` option indicates the root of your React Native apps – this will be our `ReactComponents` directory containing the single `index.ios.js` file. This running server will package up the `index.ios.bundle` file accessible via `http://localhost:8081/index.ios.bundle`.
 
+## Update App Transport Security
+
+On iOS 9 and above the app won't be a able to connect over http to localhost unless specifically told so. See this thread for alternatives and instructions: http://stackoverflow.com/questions/31254725/transport-security-has-blocked-a-cleartext-http.
```

Comment:
@andrewsardone agreed wholeheartedly.

@hfossli I think in the docs, we should recommend what Andrew says -- to register an exception for localhost.

@andrewsardone or @hfossli - either of you want to work on another PR to add this to the CLI?